docs/planification.rst

```diff
@@ -2,9 +2,10 @@
 Package Structure
 =================
 
-Here is `xhydro`'s structure as of v0.3.3 (2024-02-22), with the planned or discussed additions for the future.
+Here is `xhydro`'s structure as of 2024-06-07, with the planned or discussed additions for the future.
 
 Testing data can be found here: https://github.com/hydrologie/xhydro-testdata
+The LSTM module can be found here: https://github.com/hydrologie/xhydro-lstm. It is not included in this package to avoid dependencies on `tensorflow`.
```

Comment:
```suggestion

The LSTM module can be found here: https://github.com/hydrologie/xhydro-lstm. It is not included in this package to avoid dependencies on `tensorflow`.
```